src/api/app/views/webui2/webui/user/_breadcrumb_items.html.haml

```diff
@@ -15,3 +15,11 @@
   - if action_name == 'show'
     %li.breadcrumb-item.active{ 'aria-current' => 'page' }
       Home of #{@displayed_user}
+- when 'register_user'
+  = render partial: 'webui/configuration/breadcrumb_items'
+  %li.breadcrumb-item.active{ 'aria-current' => 'page' }
+    Create User
```

Comment:
We're missing one level in the breadcrumbs. Right now, it is `Home(link) / Configuration(link) / Create User(plain text)`, but it should be `Home(link) / Configuration(link) / Manage Users(link) / Create User(plain text)`